### PR TITLE
LSR-1026 Liferay Screens Xamarin: samples

### PR DESCRIPTION
**Pull request by dgarciasarai**
Create demo apps with auth screenlets: LoginScreenlet, ForgotPasswordScreenlet and SignUpScreenlet.
- Showcase-Android
- Showcase-iOS

### CODE REVIEW

File: xamarin/Samples/Showcase-Android/Assets/AboutAssets.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Any raw assets you want to be deployed with your application can be placed in
```
Comment (nhpatt) [inline xamarin/Samples/Showcase-Android/Assets/AboutAssets.txt]: Why a .txt and not a .md?

**Comment by nhpatt** [comment]
Looks OK

File: xamarin/Samples/Showcase-Android/Assets/AboutAssets.txt (hunk 1 above)
Comment (dgarciasarai) [inline xamarin/Samples/Showcase-Android/Assets/AboutAssets.txt]: This file is automatically generated when you create a Xamarin solution. So...It's only information about how to use Assets. Should we keep this file or not?

**Comment by nhpatt** [comment]
👍

**Pull request merged**